core/src/main/java/com/amplifyframework/auth/AuthCategoryBehavior.java

```diff
@@ -34,7 +34,7 @@
      * Creates a new user account with the specified username and password.
      * Can also pass in user attributes to associate with the user through
      * the options object.
-     * @param username This can be a typical username, email, phone number, etc. depending on the configuration
+     * @param username A login identifier e.g. `superdog22`; or an email/phone number, depending on configuration
```

Comment:
While more accurate, I think that makes it much more difficult to understand. As a developer, in a sign in method, I expect a username and password parameter. And it's not a big leap to understand I can also have email/phone as my username. But any parameter name with "identifier" in it I'm going to be thinking it's a User ID of some sort.

Reply:
Sure. That take makes sense to me, too.

Reply:
I do like the expanded examples, though. Could probably split the difference and end up with something like:
> This can be a username in the ordinary sense ("superdog22"), or an email address ("superdog22@example.com"), or a phone number ("+12125551212").
For extra credit, each could include a match regex... :-)